scripts/dictionary_tools.py

```
@@ -0,0 +1,589 @@
+
+    # .dict
+    dict_bytes = bytearray()
+    dict_offsets: list[tuple[int, int]] = []
+    for defn in merged_defs:
+        dict_offsets.append((len(dict_bytes), len(defn)))
+        dict_bytes += defn
+    (output / f"{out_stem}.dict").write_bytes(dict_bytes)
+
+    # .idx
+    idx_parts: list[bytes] = []
+    for word, (offset, size) in zip(sorted_words, dict_offsets):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)